prow/config.yaml

```diff
@@ -873,6 +873,60 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  - name: pull-kubernetes-node-e2e-prow
+    agent: kubernetes
+    branches:
+    - release-1.6
+    always_run: false
+    skip_report: false
+    max_concurrency: 1
+    context: pull-kubernetes-node-e2e-prow
+    rerun_command: "/test pull-kubernetes-node-e2e-prow"
+    trigger: "(?m)^/test pull-kubernetes-node-e2e-prow,?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20171008-cd94f30a-1.6
+        args:
+        - --root=/go/src
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        - name: USER
+          value: prow
+        - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+          value: /etc/ssh-key-secret/ssh-private
+        - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+          value: /etc/ssh-key-secret/ssh-public
+        volumeMounts:
+        - mountPath: /etc/service-account
+          name: service
+          readOnly: true
+        - mountPath: /etc/ssh-key-secret
+          name: ssh
+          readOnly: true
+        - name: cache-ssd
+          mountPath: /root/.cache
+        ports:
+        - containerPort: 9999
```

Comment:
If it doesn't use bazel-build do we need these container ports?

Reply:
git cache :-(

Reply:
right, can't that be shared safely?

Reply:
we probably want to have some solution there, or else we assume `max_concurrency` for all presubmit job = num of build nodes